nav2_bt_navigator/behavior_trees/navigate_w_replanning.xml

```diff
@@ -11,7 +11,7 @@
           <ComputePathToPose goal="{goal}"/>
         </Fallback>
       </RateController>
-      <FollowPath path="{path}"/>
+      <ComputeControl path="{path}" controller_name="FollowPath"/>
```

Comment:
If we follow my suggestion, this would then become `<MoveBase path="{path}" motion_type="FollowPath"/>`, IMHO that is more clear.

Reply:
See above, would you be happy with changing back to `FollowPath`?